docs/src/usage/analysis.md

```diff
@@ -157,6 +157,38 @@ save("aviz_scenario.png", tf)  # save the figure to a file
 
 ![Scenarios with subplots](/ADRIA.jl/dev/assets/imgs/aviz_scenario.png?raw=true "Scenarios with subplots")
 
+### Intervention location selection - visualisation
+
+Plot spatial colormaps of site selection frequencies and other available site selection metrics.
+
+```julia
+
+# Calculate frequencies with which each site was selected at each rank
+rank_freq = ranks_to_frequencies(
+    rs.ranks[intervention=1];
+    agg_func=x -> dropdims(sum(x; dims=:timesteps); dims=:timesteps),
+)
```

Comment:
I thought we agreed to use named dimensions, e.g., `intervention=:seed` - or is this in a separate PR?

Reply:
Ideally yes but the ranks storage in ResultSet is created to have interventions named 1 and 2. I changed it in standalone site selection to be consistent with this. I imagine this will change when we move to YAXarrays (?), but for now  the dimension names are created automatically: https://github.com/open-AIMS/ADRIA.jl/blob/21349226ad68a0d00b8f5fea6adbf51efd97f8be/src/io/ResultSet.jl#L75-L79

Reply:
Yes, sounds reasonable.